framework/src/media/codec/audio_encoder.cpp

```diff
@@ -185,7 +186,7 @@ int audio_encoder_getframe(audio_encoder_p encoder, void *data, size_t len)
 	}
 }
 
-int audio_encoder_init(audio_encoder_p encoder, size_t rbuf_size, audio_type_t audio_type, void *enc_ext)
+int audio_encoder_init(audio_encoder_p encoder, size_t rbuf_size, int audio_type, void *enc_ext)
```

Comment:
Previous is correct, isn't it?

Reply:
`audio_type_t` is defined in MediaType.h media namespace.
now I want to keep all APIs of `codec`  in C style, so I change it to `int`.